core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -297,7 +297,7 @@ protected void startServingRPCServer() {
       if (mTServerSocket != null) {
         mTServerSocket.close();
       }
-      mTServerSocket = new TServerSocket(mRpcAddress,
+      mTServerSocket = new TServerSocket(NetworkAddressUtils.getBindAddress(ServiceType.MASTER_RPC),
```

Comment:
- can you put javadoc here indicating it is necessary to use bind address?
- not sure if it is feasible to have a unit test catching this? we could leave this as a TODO